### ISSUE TITLE
Administraton page - error 500

### ISSUE THREAD

**Issue by hikerus** [pixelfed/pixelfed#2832]
Just updated to the recent version of Pixelfed. 

Now I get an error 500 when I want to get to administration menu (https://domain.tld/i/admin/dashboard) In php log I didn't find anything...

**Comment by dansup** [comment]
Can you check the logs in `storage/logs/laravel.log`? Can you access the https://domain.tld/i/admin/diagnostics/home page?

**Comment by tschuxxi** [comment]
I don't get a error 500 but the Dashboard is not loading. When I check the logs I get the following.

```
SQLSTATE[42803]: Grouping error: 7 ERROR: column "statuses.uri" must appear in the GROUP BY clause or be used in an aggregate function
LINE 1: ...created_at) as date, count(statuses.id) as count, statuses.*...
^ (SQL: select Date(created_at) as date, count(statuses.id) as count, statuses.* from "statuses" where "created_at" >= 2021-07-01 06:46:28 and "statuses"."deleted_at" is null group by "statuses"."id", Date(created_at) order by "created_at" desc)
```

**Comment by hikerus** [comment]
After last git upgrade error 500 has gone and the administration page can be reached.

**Comment by ThomasLewisNHTB** [comment]
I fixed this by editing app/Services/AdminStatsService.php lines 71, 72, and 89. Here are the modified lines in order. It seems to work as originally intended based on the graph displayed on the admin page.

```
                        $gb = config('database.default') == 'pgsql' ? ['statuses.id','statuses.created_at', DB::raw('Date(created_at)')] : DB::raw('Date(created_at)');
                        $s = Status::selectRaw('Date(created_at) as date, count(statuses.id) as count, statuses.id')
...

                        $s = Status::selectRaw('Date(created_at) as date, count(statuses.id) as count, statuses.id')
```

**Comment by lapineige** [comment]
Was that "fix" included in 0.11.1 ? (released in september https://github.com/pixelfed/pixelfed/releases/tag/v0.11.1)

We experience that issue on fresh Yunohost installations https://github.com/YunoHost-Apps/pixelfed_ynh/pull/156

**Comment by lapineige** [comment]
If not, how do we know in which version this fix is/will be included ?
